client/gutenberg/editor/calypsoify-iframe.tsx

```diff
@@ -77,6 +77,7 @@ interface Props {
 	postType: T.PostType;
 	editorType: 'site' | 'post'; // Note: a page or other CPT is a type of post.
 	pressThis: any;
+	anchorFmData: any;
```

Comment:
We could probably define this type, right?

Reply:
I think you're right 👍